src/JustEat.StatsD/PooledUdpTransport.cs

```diff
@@ -51,6 +53,29 @@ public void Send(string metric)
             pool.Push(socket);
         }
 
+        public void Send(ArraySegment<byte> metric)
+        {
+            if (metric.Count == 0)
```

Comment:
Same question regarding if it can be `null`.

Reply:
Array which it holds can (if you construct ArraySegment with default constructor) or just use `default(ArraySegment<byte>)`. Will add checks